google/ggrc-core
# GGRC-3453 latest workflow-acl into dev

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] #7156 
- [x] #7180 

To Prevent conflicts from release to dev, this PR has indirect dependecies:

- [x] https://github.com/google/ggrc-core/pull/7188
- [x] https://github.com/google/ggrc-core/pull/7166
- [x] #7199 Merge release branch to dev (PR not opened yet)
- [x] #7202 

# Issue description

This big PR replaces deprecated UserRole usage for Workflow with ACL. UserRoles were migrated to ACL. 
Next roles were replaced:
- WorkflowOwner (Workflow Manager on UI) => Admin
- WorkflowMember => Workflow Member
On UI - People tab was removed. Instead of using it user should setup people and assign roles for them on Workflow info pane / Workflow Create page.
All the other stuff should be the same.

# Steps to test the changes
1. Create Workflow.
2. Add some people as Admin or Workflow Member to it.
3. Create TaskGroup.
4. Check DB that Admin, Admin Mapped, Workflow Member, Workflow Member Mapped were created by above actions. Mapped roles used for permissions propagation.

# Solution description

Added Admin, Admin Mapped, Workflow Member, Workflow Member Mapped roles into DB. Migrated UserRoles into ACL records with roles described above.

# Sanity checklist

# Migration checklist

# PR Review checklist
